docs/graphql/core/api-reference/schema-metadata-api/index.rst

```diff
@@ -104,6 +104,11 @@ The various types of queries are listed in the following table:
      - 2
      - Add a table/view with configuration
 
+   * - :ref:`set_table_configuration <set_table_configuration>`
+     - :ref:`set_table_configuration_args <set_table_configuration_syntax>`
+     - 1
+     - Set table configuration of an already tracked table
```

Comment:
The name “configuration” here is also kind of vague. We have other options on tables, such as `is_enum`, and those things are really just as much “configuration” as the things this API supports. I think a better term would be *customization*, as all these things seem customization-related. So what about calling this `set_table_customization`, instead?

Reply:
I've refactored the API to `set_table_customization`.